code/noRAM/app/src/main/java/com/example/noram/AdminImagesFragment.java

```diff
@@ -100,7 +100,10 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
                     @Override
                     public void onSuccess(ListResult listResult) {
                         for(StorageReference item : listResult.getItems()){
-                            addPhotoToGrid(item, true);
+                            // add photo if it's not a setting photo
+                            if(!item.getPath().contains("Cupcake")){

```

Comment:
I think we also want to include lowercase cupcake to catch one of the other photos in cloud storage right now. Also not sure if we want to exclude default profile pictures here, that would need to also exclude "-default"